es-ds-docs/pages/molecules/video.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+const propTableRows = [
+    ['alt-text', 'String', 'n/a', 'Required.'],
```

Comment:
These props need a complete description, what would someone using `EsVideo` for the first time need to know?